<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MouseWindow</class>
 <widget class="QMainWindow" name="MouseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaeRemote</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.png</normaloff>logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(mouseui.png);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-image: url(mouseui.png);
background-color: transparent;</string>
   </property>
   <widget class="QPushButton" name="rightMouseButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>381</width>
      <height>161</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="whatsThis">
     <string>Right click button....</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(nopic.png);
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>right button</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftMouseButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>411</width>
      <height>171</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="whatsThis">
     <string>Left click button...</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(nopic.png);
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>left button</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lockButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>310</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(nopic.png);
background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>freeze.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>110</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="toggleButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(nopic.png);
background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>openlock_small.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QFrame" name="scrollFrame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>461</width>
      <height>101</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>Scroll bar and middle click button...</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(nopic.png);
background-color: transparent;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>leftMouseButton</sender>
   <signal>clicked()</signal>
   <receiver>MouseWindow</receiver>
   <slot>leftMouseButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toggleButton</sender>
   <signal>clicked()</signal>
   <receiver>MouseWindow</receiver>
   <slot>toggleButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>695</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lockButton</sender>
   <signal>pressed()</signal>
   <receiver>MouseWindow</receiver>
   <slot>lockButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>682</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lockButton</sender>
   <signal>released()</signal>
   <receiver>MouseWindow</receiver>
   <slot>lockButtonReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>684</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>rightMouseButtonClicked()</slot>
  <slot>leftMouseButtonClicked()</slot>
  <slot>toggleButtonClicked()</slot>
  <slot>lockButtonPressed()</slot>
  <slot>lockButtonReleased()</slot>
 </slots>
</ui>
